commercetools-models/src/main/java/io/sphere/sdk/orders/LineItemImportDraftBuilder.java

```diff
@@ -23,6 +24,8 @@
     private Reference<Channel> supplyChannel;
     @Nullable
     private TaxRate taxRate;
+    @Nullable
+    private CustomFieldsDraft custom;
 
     private LineItemImportDraftBuilder(final ProductVariantImportDraft variant, final Long quantity, final Price price, @Nullable final String productId, final LocalizedString name) {
```

Comment:
Sorry that I didn't see this in my first review, but we should add the `custom` as a parameter to this constructor and then initialize the `custom` field with it.